src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -112,6 +112,27 @@ maxLength switch
                 .WithVariableParameter("arg", LanguageConstants.Array, minimumCount: 1, "The array for concatenation")
                 .Build();
 
+            yield return new FunctionOverloadBuilder("cidrSubnet")
+                .WithReturnType(LanguageConstants.String)
+                .WithGenericDescription("Returns the specified subnet of a CIDR network.")
+                .WithRequiredParameter("network", LanguageConstants.String, "The string containing an IP network (CIDR format)")
+                .WithRequiredParameter("cidr", LanguageConstants.Int, "The CIDR to be used to subnet the current IPNetwork")
+                .WithRequiredParameter("subnetIndex", LanguageConstants.Int, "The index to select the subnet from the range of CIDR subnets")
+                .Build();
+
+            yield return new FunctionOverloadBuilder("cidrHost")
+                .WithReturnType(LanguageConstants.String)
+                .WithGenericDescription("Calculates the ith IP address on a network.")
+                .WithRequiredParameter("network", LanguageConstants.String, "The string containing an ip network (CIDR format)")
+                .WithRequiredParameter("num", LanguageConstants.Int, "A binary integer with no more than the number of digits remaining in the address after the given network string")
```

Comment:
>A binary integer with no more than the number of digits remaining in the address after the given network string [](http://example.com/codeflow?start=70&length=111)

A 0-based index of the usable host on the specified network. Must be less than the number of usable hosts on the specified network.